### ISSUE TITLE
[Feature] Add tables about stacking bars

### ISSUE THREAD

**Issue by linhuanxia** [apache/echarts#19710]
### What problem does this feature solve?

I wonder if I can reproduce this drawing using echarts
![微信图片_20240312221448](https://github.com/apache/echarts/assets/38238997/91c31242-8734-48a1-b60b-9c359054c02c)

### What does the proposed API look like?

Give some methods or examples，Link!

**Comment by helgasoft** [comment]
see #19250 for solution
📌 please close issue if problem solved.

**Comment by linhuanxia** [comment]
No, it is not quite the same, mine is stacked bar chart + X-axis for the table, but it seems that the space between the columns does not need to be so troublesome, difficult in how to integrate into this composite chart @helgasoft

**Comment by Ovilia** [comment]
This is not supported by default. You may use `graphic.elements` to draw lines and text.
